build.gradle

```diff
@@ -130,6 +130,16 @@ tasks.withType(FindBugs) {
   }
 }
 
+jacocoTestReport {
```

Comment:
Ok, I know what my problem was. The `jacocoTest` or `jacocoTestReport` tasks don't do anything  if no tests were run. 
```
./gradlew --daemon jacocoTest
[...]
:jacocoTestReport SKIPPED
```
I supposed their was a dependency between the two.  Adding it to `check` dependencies solved “my issue” since `check` depends on tests.